From 3beaa6cc8222ca4e564db6fd1e3540c12332f277 Mon Sep 17 00:00:00 2001
From: Ben Peled <bpeled@marvell.com>
Date: Thu, 11 Apr 2019 13:46:27 +0300
Subject: [PATCH 169/386] arm64: marvell: enable the Armada 7K/8K CPU clk
 driver

This commit makes sure the driver for the Armada 7K/8K CPU clock is
enabled.

Change-Id: I30fc50e3e84b8ad9f6b5d519d6ac0ea7b48d51c5
Signed-off-by: Gregory CLEMENT <gregory.clement@bootlin.com>
Reviewed-on: https://sj1git1.cavium.com/7225
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Nadav Haklai <Nadav.Haklai@cavium.com>
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
[RH: Original patch taken from marvell 88F3720 board support SDK 10.0-PR2003]
Signed-off-by: Ruiqiang Hao <Ruiqiang.Hao@windriver.com>
---
 arch/arm64/Kconfig.platforms | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/Kconfig.platforms b/arch/arm64/Kconfig.platforms
index b9e914452a3e..22db8105c7c9 100644
--- a/arch/arm64/Kconfig.platforms
+++ b/arch/arm64/Kconfig.platforms
@@ -121,6 +121,7 @@ config ARCH_MESON
 
 config ARCH_MVEBU
 	bool "Marvell EBU SoC Family"
+	select ARMADA_AP_CPU_CLK
 	select ARMADA_AP806_SYSCON
 	select ARMADA_AP810_SYSCON
 	select ARMADA_CP110_SYSCON
-- 
2.17.1

